src/containers/ConceptPage/conceptUtil.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { FormikContextType } from 'formik';
+import isEmpty from 'lodash/fp/isEmpty';
+import { plainTextToEditorValue, editorValueToPlainText } from '../../util/articleContentConverter';
+import { createEmbedTag } from '../../util/embedTagHelpers';
+import { NewConceptType, PatchConceptType } from '../../modules/concept/conceptApiInterfaces';
+import { SubjectType, License, ConceptType, CreateMessageType } from '../../interfaces';
+import { ConceptFormValues, ConceptFormType } from './conceptInterfaces';
+
+export const transformApiConceptToFormValues = (
+  concept: ConceptFormType,
+  subjects: SubjectType[],
+): ConceptFormValues => {
+  return {
+    id: concept.id,
+    title: concept.title || '',
+    language: concept.language,
+    updated: concept.updated,
+    updateCreated: false,
+    subjects: concept.subjectIds
+      ? subjects.filter(subject => concept.subjectIds.find(id => id === subject.id))
+      : [],
+    created: concept.created,
+    conceptContent: plainTextToEditorValue(concept.content || '', true),
+    supportedLanguages: concept.supportedLanguages || [],
+    creators: concept.copyright?.creators || [],
+    rightsholders: concept.copyright?.rightsholders || [],
+    processors: concept.copyright?.processors || [],
+    source: concept && concept.source ? concept.source : '',
+    license: concept.copyright?.license?.license || '',
+    metaImageId: concept.metaImageId,
+    metaImageAlt: concept.metaImage?.alt || '',
+    tags: concept.tags || [],
+    articles: concept.articles || [],
+    status: concept.status || {},
+    visualElementObject: concept.parsedVisualElement || {},
+  };
```

Comment:
Jeg set at denne er brukt i ConceptForm på en concept som kommer fra conceptApi og navnet på funksjonen sier også at den skal gjøre om fra ConceptApi til ConceptForm, men typene till ConceptApi stemmer ikke helt: 
* `title` er et objekt fra api'et, så å sette (objekt || string) blir litt feil, burde kanskje bruke `convertFieldWithFallback()`?
* `language` feltet kommer ikke fra api'et, hvis du vil få tak i `language` feltet må du enten kikke inn i concept.supportedLanguages eller individuelt i språk-objekter fra api'et som title eller content.
* `metaImageId` feltet finnes ikke i ConceptApi typen,  metaImage objektet består av (alt, url, language) feltene
* `articles` feltet finnes ikke, kun `articleIds`

Ta en kikk på  `/model/api/Concept.scala` i ConceptApi for å se hele typen som kommer fra ConceptApi til frontenden

Reply:
Og jeg ville rename `concept: ConceptFormType,` om til `concept: ConceptApiType` slik at det er mer tydelig hva typen skal være

Reply:
Det er flere typer her.
ApiConceptType er den som er fra apiet.
Den blir oversatt til ConceptType i conceptApiUtil
ConceptType er nøyaktig det samme som fra apiet, bare med flata ut språkobjekter

ConceptFormType er ConceptType + articles som er henta separat.
Formet får ConceptFormType, som så må gjøres om til formValues Som er verdiene formik jobber med. Det er transformeringen du ser her.

Reply:
Det er altså ikke apitypen, det er en flata ut og expanda api type.

Reply:
Alle apikallene har en innebygget transform nå, så ApiConceptType er ikke ibruk noe sted ellers i frontendkoden

Reply:
Ah skjønner, da er jeg med, da er kommentaren min invalid.